packages/web3/test/black_box/test/web3-eth-accounts/hashMessage.test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// TODO For some reason when running this test with a WebSocket provider
+// the test takes a long time to run afterAll
+import Web3 from 'web3';
+import {
+	closeOpenConnection,
+	getSystemTestProvider,
+	isWs,
+} from 'web3/test/shared_fixtures/system_tests_utils';
+
+jest.setTimeout(15000);
```

Comment:
For only this test and `packages/web3/test/black_box/test/web3-eth-accounts/create.test.ts`, the `if (isWs) await closeOpenConnection(web3);` in `afterAll` takes longer to complete and surpasses the default timeout for Jest